### ISSUE TITLE
Update numexpr

### ISSUE THREAD

**Issue by danielballan** [nsls2-conda-envs/nsls2-collection-tiled#22]
With @jgoodrich I saw this:

```
/nsls2/conda/envs/2023-3.1-py310-tiled/lib/python3.10/site-packages/pandas/core/computation/expressions.py:21: UserWarning: Pandas requires version '2.8.0' or newer of 'numexpr' (version '2.7.3' currently installed).
  from pandas.core.computation.check import NUMEXPR_INSTALLED
```

The first 2.8.x release [was in December 2021](https://pypi.org/project/numexpr/#history). Something must be holding numexpr back. Does this ring a bell, @mrakitin?

**Comment by mrakitin** [comment]
We should try a newer pin and see what happens.

**Comment by mrakitin** [comment]
It's addressed via #24.
